partition_table.go

```diff
@@ -239,21 +251,24 @@ func (p *PartitionTable) load(ctx context.Context, stopAfterCatchup bool) (rerr
 		return
 	}
 
+	p.log.Printf("Offset stored locally: %d", localOffset)
+
 	loadOffset, hwm, err := p.findOffsetToLoad(localOffset)
 	if err != nil {
 		errs.Collect(err)
 		return
 	}
 
 	if localOffset >= hwm {
-		errs.Collect(fmt.Errorf("local offset is higher than partition offset. topic %s, partition %d, hwm %d, local offset %d", p.topic, p.partition, partitionHwm, localOffset))
+		errs.Collect(fmt.Errorf("local offset is higher than partition offset. topic %s, partition %d, hwm %d, local offset %d", p.topic, p.partition, hwm, localOffset))
```

Comment:
let's add an extra step how to fix this, like `Probably the kafka topic was recreated but the processor still has it's old local cache`